src/sota_tools/CMakeLists.txt

```diff
@@ -11,7 +11,12 @@ set (GARAGE_PUSH_SRCS
   ostree_object.cc
   request_pool.cc
   treehub_server.cc
+  ${PROJECT_SOURCE_DIR}/src/utils.cc
+  ${PROJECT_SOURCE_DIR}/src/logger.cc
+  ${PROJECT_SOURCE_DIR}/third_party/jsoncpp/jsoncpp.cpp
 )
+set_source_files_properties(${PROJECT_SOURCE_DIR}/third_party/jsoncpp/jsoncpp.cpp PROPERTIES COMPILE_FLAGS -w)
+
```

Comment:
I'm not keen on having a CMakeLists pull files from another random directory--can't we use a library (libaktualizr) to share this between projects, rather than having one C++ file that is built into several binaries?

Reply:
Yes, we can